### ISSUE TITLE
Some questions about the experimental results of the CRC dataset

### ISSUE THREAD

**Issue by junjianli106** [srinidhiPY/SSL_CR_Histo#6]
Thank you for your excellent paper and open source code. I have some questions about the experimental results of NCT-CRC. 

1. **The MoCo + CR approach obtains a new state-of-the-art result with an Acc of 0.990, weighted F-1 score of 0.953 and a macro AUC of 0.997, compared to the previous method ( Kather et al., 2019 ) which obtained an Acc of 0.943.**  However, using random initialization can get 97.2% acc with 10% training data in Table 5, which is also much higher than 0.943 of (kather et al., 2019), random initialization can also get high ACC, did I miss something? 

> Table 5 presents the overall Acc and weighted F 1 score ( F 1 ) for classification of 9 colorectal tissue classes using different methodologies. On this dataset, the MoCo + CR approach obtains a new state-of-the-art result with an Acc of 0.990, weighted F-1 score of 0.953 and a macro AUC of 0.997, compared to the previous method ( Kather et al., 2019 ) which obtained an Acc of 0.943.

2. When I train the CRC dataset, the difference between my weighted F1 and ACC was not as great as yours（Acc :0.990, weighted F-1: 0.953）, for example, ACC:0.9400, weight-F1:0.9399 , did I miss something?

**Comment by srinidhiPY** [comment]
Hi Junjian Li,

Thank you for showing interest in our work.

Here is the answer to your questions:

1) Yes, random initialization also gets higher Accuracy compared to Kather
et al, as per our experiments. Since, this task was one of the easiest due
to its well curated dataset, most methods perform eqUally well.

2) did you use the same hyper-parameters as reported in our paper ? You can
reproduce the results from our code and let me know if you have any
questions.
